MdeModulePkg/Core/Dxe/UnitTest/MemoryProtectionUnitTestHost.inf

```diff
@@ -0,0 +1,183 @@
+## @file

```

Comment:
Looks like this is a bad merge, this should be dropped, right?

Reply:
yeah, thought i did that locally, but I guess not haha